pylint_venv.py

```diff
@@ -65,11 +65,14 @@ def detect_venv():
         if venv:
             return venv
 
-    # Check if a local ".venv" folder exists
+    # Check if a virtualenv directory exists (.venv by default, or alternative
+    # paths given by environment variable)
+    venv_paths = os.getenv("PYLINT_VENV_PATH", ".venv").split(":")
```

Comment:
Note: This does not support any directories where the name itself contains a colon (`:`). However, I think it is fine because the same is true for Bash's `$PATH` variable and I hope no-one uses colons in the venv directory names (or any other directory names for that matter).